frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineRunCount.tsx

```diff
@@ -37,15 +37,16 @@ const PipelineRunCount: React.FC<PipelineMetricsGraphProps> = ({
   });
   const pipelineRunResultData = pipelineRunResult?.data?.result ?? [];
 
+  React.useEffect(() => {
```

Comment:
to avoid react warnings for bad setState calls, used useEffect in all chart components.